embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/config/behavior/SessionBehavior.kt

```diff
@@ -142,6 +142,14 @@ internal class SessionBehavior(
     fun shouldSendFullForErrorLog() =
         getFullSessionEvents().contains(SessionGatingKeys.FULL_SESSION_ERROR_LOGS)
 
+    /**
+     * Checks whether the session start message should be sent to the API or not.
+     */
+    fun isStartMessageEnabled(): Boolean {
+        return thresholdCheck.isBehaviorEnabled(remote?.sessionConfig?.pctStartMessageEnabled)
+            ?: true
```

Comment:
What do you think about getting wild and defaulting to `false` 😈

Reply:
I'd be happy with that - probably something to run past @fnewberg when he's back.

Reply:
Lets just do it this way and we can turn this on by default in 6.2. Given that I want to start the release today, I'm just going to merge this for you so we can get it as part of 6.1. Since it's default off right now, it should be fine. :-)

Reply:
@fractalwrench did you talk with pablo about just turning this off? i'm pretty sure we can just remove it entirely rather than adding a config for it

Reply:
IIRC our previous conversations suggested adding a config flag to disable this in case anything unforeseen did go wrong. We can switch the default or remove the flag entirely once we've validated that is true

Pablo did not have any concerns about not sending the start message